acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -50,7 +51,14 @@ describe('Critical user path(s)', () => {
 
     it('Happy path - Adult', async() => {
         try {
+            // start
             await clickContinueButton(1);
+
+            // who-do-you-work-for
+            await page.waitForSelector(ORGANISATION_INPUT);
+            await page.$eval(ORGANISATION_INPUT, (element) => {
+                element.value = 'Barnardos';
```

Comment:
This suggests ms-organisations have the format of first letter uppercase. Is that consistent throughout the app? E.g. you have england as first letter lowercase

I’d probably stick to lowercase